doc/release/1.13.0-notes.rst

```diff
@@ -29,9 +32,8 @@ Future Changes
 Compatibility notes
 ===================
 
-
 Error type changes
-~~~~~~~~~~~~~~~~~~
```

Comment:
1.11, 1.10, 1.9, 1.8, and probably more, all used tildes for these headings. 1.12 is the outlier, not 1.13